pyHS100/__init__.py

```diff
@@ -1 +1 @@
-
+from pyHS100.pyHS100 import SmartPlug
```

Comment:
False alarm. import pyHS100 allows using pyHS100.SmartPlug as well as from pyHS100 import SmartPlug for library users. Can be reverted though, if from pyHS100.pyHS100 import SmartPlug is more wanted.